src/EditorFeatures/Core/ExternalAccess/EditorConfig/Features/IEditorConfigQuickInfoService.cs

```diff
@@ -0,0 +1,18 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+#nullable disable
+
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.CodeAnalysis;
+using Microsoft.CodeAnalysis.Host;
+
+namespace Microsoft.CodeAnalysis.ExternalAccess.EditorConfig.Features
+{
+    internal interface IEditorConfigQuickInfoService : ILanguageService
+    {
+        Task<EditorConfigQuickInfo> GetQuickInfoAsync(TextDocument document, int position, CancellationToken cancellationToken);
+    }
```

Comment:
surprised we're not making a subclass of QuickInfoService for the editorconfig case.  note: we would need to revise that API to support TextDocument, not just Document.  But i think that would be totally clean to do.

Reply:
yeah this is certainly one way we could do this.  But lets put this in the next PR (since it looks its unused currently).